src/Nethermind/Nethermind.Clique.Test/CliqueBlockProducerTests.cs

```diff
@@ -195,6 +193,8 @@ public On CreateNode(PrivateKey privateKey, bool withGenesisAlreadyProcessed = f
                     MainnetSpecProvider.Instance, 
                     _cliqueConfig,
                     nodeLogManager);
+
+                var suggester = new ProducedBlockSuggester(blockTree, blockProducer);
```

Comment:
Not used?

Reply:
ProducedBlockSuggester hooks to events, I can remove the variable which is not used, but prefer to have it there.